test/Test/Toml/Parsing/Unit.hs

```diff
@@ -396,49 +396,39 @@ tableSpecs :: Spec
 tableSpecs = do
     describe "tableP" $ do
         it "can parse a TOML table" $
-          parseTable "[table-1]\nkey1 = \"some string\"\nkey2 = 123"
-                (makeKey ["table-1"], tomlFromKeyVal [str, int])
+          parseTable "key1 = \"some string\"\nkey2 = 123"
+                (tomlFromKeyVal [str, int])
         it "can parse an empty TOML table" $
-            parseTable "[table]" (makeKey ["table"], tomlFromKeyVal [])
-        it "allows the name of the table to be any valid TOML key" $ do
-            parseTable "[dog.\"tater.man\"]"
-                (makeKey ["dog", dquote "tater.man"], tomlFromKeyVal [])
-            parseTable "[j.\"ʞ\".'l']"
-                (makeKey ["j", dquote "ʞ", squote "l"], tomlFromKeyVal [])
```

Comment:
I don't quite understand why so many tests were deleted :thinking: They seem useful because they test unicode symbols. It's not obvious from the implementation, why these tests are redundant now.

Reply:
I had to remove them because I've refactored the key parsing out of the table parsing function. Those exact same tests on unicode keys are tested somewhere else, so they were redundant anyway.
I will add more tests like you suggested though. There are also some corner cases that I should add.